### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by cht8687** [tj/frontend-boilerplate#9]
Any plans to include tests?

**Comment by tj** [comment]
Yeah maybe, I've never been a fan of integration tests for front-end but unit tests for dumb components would be good.

**Comment by sotojuan** [comment]
Testing Redux stuff is also very easy.

**Comment by tj** [comment]
yep, I definitely see that as a huge advantage of redux

**Comment by thebigredgeek** [comment]
Are we cool with Mocha?  If so I will submit PR tomorrow

**Comment by tj** [comment]
doesn't matter to me, I don't really test front-end UI related stuff, it either works while I'm starring at it all day or it doesn't.

**Comment by thebigredgeek** [comment]
Cool.  Mocha it is
